config/config.go

```diff
@@ -81,6 +83,9 @@ type Config struct {
 	Gcp Gcp
 }
 
+// Allowed StateBackend Types
+var StateBackendTypes = []string{"local", "s3", "azblob", "gs"}
```

Comment:
The last option should be `gcs`

Reply:
To make this more easier to understand could we have to arrays 
```go
remoteStateBackendTypes = []string{"s3", "azblob", "gcs"}
localStateBackendTypes = []string{"local"}

StateBackendTypess = append(remoteStateBackendTypes, localStateBackendTypes...)
```
that'll make referencing remote and local states from within the code base easier

Reply:
The last option is "gs" because all the options just get passed as prefixes in creating bucket links... except you want to add extra logic to change gcs to gs

Reply:
All right no issues for that we could leave it that way